node/src/actors/chain_manager/mod.rs

```diff
@@ -2124,14 +2124,18 @@ impl ChainManager {
 
     /// Return the value of the version field for a block in this epoch
     fn tapi_signals_mask(&self, epoch: Epoch) -> u32 {
-        let Tapi { oppose_wip0014 } = &self.tapi;
+        let Tapi {
+            oppose_wip0014,
```

Comment:
if are the same activation, should not be the same oppose flag?

Reply:
It's the illusion of choice

Reply:
🤦🏽